palak/src/pages/Front.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+// import Button from "../components/Button";
```

Comment:
If some comment is not required for any references or future use try to remove it.